sparse/usr/lib/qt5/plugins/feedback/qtfeedback-cancro.ini

```diff
@@ -1,134 +1,18 @@
-#
-# _TYPE =	[rumble|periodic], no default, mandatory parameter
-# _DIRECTION = 	[forward|reverse], defaults to forward
-# _DURATION = 	[0,65535], milliseconds, defaults to NGF_DEFAULT_DURATION
-# _DELAY = 	[0,65535], milliseconds before starting playback, defaults to 0
-#
-# _REPEAT =	[1,4294967295], how many times to repeat effect, defaults to 1
-#		Please note that if the event has "sound.repeat" enabled,
-#		the effect is repeated until stop signal regardless of _REPEAT
-#		value.
-#
-# - Type specific parameters for rumble effects:
-#
-# _MAGNITUDE =	[0,65535], defaults to NGF_DEFAULT_RMAGNITUDE
-#
-# -Type specific parameters for periodic effects
-#
-# _WAVEFORM =	[sine|triangle|square], defaults to sine
-# _PERIOD =	[0,65535], defaults to 0
-# _MAGNITUDE =	[0,32767], defaults to NGF_DEFAULT_PMAGNITUDE
-# _OFFSET =	[0,32767], defaults to 0
-# _PHASE =	[0,65535], in milliseconds, defaults to 0
-# _ATTACK =	[0,65535], in milliseconds, defaults to 0
-# _ALEVEL =	[0,32767], defaults to 0
-# _FADE =	[0,65535], in milliseconds, defaults to 0
-# _FLEVEL =	[0,32767], defaults to 0
-
-NGF_SHORT_TYPE = rumble
-NGF_SHORT_DURATION = 240
-NGF_SHORT_DELAY = 0
-NGF_SHORT_MAGNITUDE = 27000
-
-NGF_LONG_TYPE = periodic
-NGF_LONG_WAVEFORM = sine
-NGF_LONG_DURATION = 850
-NGF_LONG_PERIOD = 150
-NGF_LONG_MAGNITUDE = 15383
-NGF_LONG_ATTACK = 0
-NGF_LONG_ALEVEL = 0
-NGF_LONG_FADE = 180
-NGF_LONG_FLEVEL = 0
-
-NGF_STRONG_TYPE = rumble
-NGF_STRONG_DURATION = 320
-NGF_STRONG_DELAY = 0
-NGF_STRONG_MAGNITUDE = 65535
-
-NGF_BATTERYLOW_TYPE = periodic
-NGF_BATTERYLOW_WAVEFORM = sine
-NGF_BATTERYLOW_DURATION = 120
-NGF_BATTERYLOW_REPEAT = 2
-NGF_BATTERYLOW_DELAY = 40
-NGF_BATTERYLOW_PERIOD = 150
-NGF_BATTERYLOW_MAGNITUDE = 19383
-NGF_BATTERYLOW_OFFSET = 0
-NGF_BATTERYLOW_PHASE = 0
-NGF_BATTERYLOW_DIRECTION = forward
-NGF_BATTERYLOW_ATTACK = 0
-NGF_BATTERYLOW_ALEVEL = 0
-NGF_BATTERYLOW_FADE = 180
-NGF_BATTERYLOW_FLEVEL = 0
-
-NGF_SMS_TYPE = periodic
-NGF_SMS_WAVEFORM = sine
-NGF_SMS_DURATION = 240
-NGF_SMS_REPEAT = 2
-NGF_SMS_DELAY = 140
-NGF_SMS_PERIOD = 50
-NGF_SMS_MAGNITUDE = 27000
-NGF_SMS_OFFSET = 0
-NGF_SMS_PHASE = 0
-NGF_SMS_DIRECTION = forward
-NGF_SMS_ATTACK = 0
-NGF_SMS_ALEVEL = 0
-NGF_SMS_FADE = 80
-NGF_SMS_FLEVEL = 0
-
-NGF_RINGTONE_TYPE = periodic
-NGF_RINGTONE_WAVEFORM = sine
-NGF_RINGTONE_DURATION = 2400
-NGF_RINGTONE_DELAY = 400
-NGF_RINGTONE_PERIOD = 100
-NGF_RINGTONE_MAGNITUDE = 16383
-NGF_RINGTONE_OFFSET = 0
-NGF_RINGTONE_PHASE = 0
-NGF_RINGTONE_DIRECTION = forward
-NGF_RINGTONE_ATTACK = 500
-NGF_RINGTONE_ALEVEL = 2560
-NGF_RINGTONE_FADE = 300
-NGF_RINGTONE_FLEVEL = 4096
-
-NGF_CLOCK_TYPE = periodic
-NGF_CLOCK_WAVEFORM = sine
-NGF_CLOCK_DURATION = 4000
-NGF_CLOCK_DELAY = 500
-NGF_CLOCK_PERIOD = 100
-NGF_CLOCK_MAGNITUDE = 8000
-NGF_CLOCK_OFFSET = 0
-NGF_CLOCK_PHASE = 0
-NGF_CLOCK_DIRECTION = forward
-NGF_CLOCK_ATTACK = 1000
-NGF_CLOCK_ALEVEL = 0
-NGF_CLOCK_FADE = 400
-NGF_CLOCK_FLEVEL = 0
-
+ACTUATOR_SPIN_UP=2
+ACTUATOR_SPIN_DOWN=4
```

Comment:
I remember the vibrator settings were different between Mi3 and Mi4. So can someone test these changes on both these devices before we merge them?